ui/src/app/features/sign-up-form/sign-up-form.component.html

```diff
@@ -74,6 +74,12 @@
           required
         />
       </mat-form-field>
+      <mat-checkbox formControlName="isAllowStatistics" class="sign-up-full-width" *ngIf="isDemoPageAvailable$ | async">
+        <small
+          >Allow to send anonymous statistics for product <br />
```

Comment:
please move wording to the translations file

Reply:
done